.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "libs/providers/flagd/schemas"]
 	path = libs/providers/flagd/schemas
-	url = https://github.com/open-feature/schemas
+	url = https://github.com/open-feature/schemas.git
+[submodule "libs/providers/flagd-web/schemas"]
```

Comment:
Would it make sense to consolidate this to a single location? Perhaps something like a top level schemas directory?

Reply:
Maybe, but this also allows us to update them separately, which has it's own advantages. If we want to do it, I'm OK with that, but I'd rather not do it with this PR.

Reply:
Fair point, this is good for now.